ui/main/src/app/business/templateGateway/cardTemplateGateway.ts

```
@@ -0,0 +1,213 @@
+    public static setTemplateListenerForTemplateRenderingComplete(listener: Function) {
+        CardTemplateGateway._functionToSendTemplateRenderingCompleteToTemplate = listener;
+    }
+
+    public static setUserAllowedToRespond(userAllowedToRespond: boolean) {
+        CardTemplateGateway._userAllowedToRespond = userAllowedToRespond;
+    }
+
+    public static setUserMemberOfAnEntityRequiredToRespond(userMemberOfAnEntityRequiredToRespond: boolean) {
+        CardTemplateGateway._userMemberOfAnEntityRequiredToRespond = userMemberOfAnEntityRequiredToRespond;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)